composer.json

```diff
@@ -1,19 +1,39 @@
 {
     "name": "ebanx/magento2-gateway-ebanx",
     "description": "Oficial EBANX Magento2 Payment Gateway",
-    "version": "1.6.6",
+    "version": "1.7.6",
```

Comment:
Shouldn't it be 1.7.0?

Reply:
Nope. We have breaks because elasticsearch ia required. Its not only a patch.

Reply:
Ow, thats right. It was a typo.